test/test_imagestreams_quickstart.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+import pytest
+from pathlib import Path
+
+from container_ci_suite.openshift import OpenShiftAPI
+from container_ci_suite.utils import get_service_image, check_variables
+test_dir = Path(os.path.abspath(os.path.dirname(__file__)))
+
+if not check_variables():
+    sys.exit(1)
+
+
+VERSION = os.getenv("SINGLE_VERSION")
+if not VERSION:
+    VERSION = "3.11"
+IMAGE_NAME = os.getenv("IMAGE_NAME")
+OS = os.getenv("OS")
+
+if VERSION == "3.11":
+    BRANCH_TO_TEST = "4.2.x"
+    DEPLOYED_PSQL_IMAGE = "quay.io/sclorg/postgresql-12-c8s:c8s"
+    IMAGE_TAG = "postgresql:12"
+    PSQL_VERSION = "12"
+else:
+    BRANCH_TO_TEST = "master"
+    DEPLOYED_PSQL_IMAGE = "quay.io/centos7/postgresql-10-centos7:centos7"
+    IMAGE_TAG = "postgresql:10"
+    PSQL_VERSION = "10"
+
+if OS == "rhel7":
```

Comment:
A dictionary (mapping) might be better than many ifs.

Reply:
Good point.

Reply:
DONE